pkg/operator/csi/csidrivercontrollerservicecontroller/csi_driver_controller_service_controller.go

```diff
@@ -89,14 +92,16 @@ func NewCSIDriverControllerServiceController(
 	deployInformer appsinformersv1.DeploymentInformer,
 	optionalConfigInformer configinformers.SharedInformerFactory,
 	recorder events.Recorder,
+	optionalDeploymentHooks ...func(*appsv1.Deployment) error,
```

Comment:
Would it make sense to either give owner of the controller a way how to poke the controller for resync (i.e. the owner has informer on secrets / config maps / whatnot and is able to enqueue "fake" event into CSIDriverControllerServiceController' queue) or pass informers to watch into the `NewCSIDriverControllerServiceController`?

I can see there is `ResyncEvery(time.Minute)` below, so this PR will work even without it.

Reply:
Would it make sense to address this in another PR?

Also, could you clarify a use case for poking the controller for resync?

Reply:
Ack